synapse/rest/client/__init__.py

```diff
@@ -1,4 +1,4 @@
-# Copyright 2015, 2016 OpenMarket Ltd
```

Comment:
Looks like we gained a year?

Anyway this should get updated to the m.org foundation.

Reply:
the __init__ in the subpackages said 2014-2016, so I widened it.
Should this be updated? I don't know the legal interpretation of that, but I thought we kept that around and M.org came in after (so the old copyright still exists).

Not that any of this makes much sense on an empty file, though...

Reply:
We update them to the matrix.org foundation for openmarket and new vector.